explainer-metrics.md

```diff
@@ -92,45 +92,26 @@ The `FormattedText` constructor's static method `format` (described in the
 internally holds all of the formatted text ready for rendering and is also the root object of
 the metrics describe in this explainer.
 
-The `FormattedText` object contains properties to retrieve the inline size and block size 
-(among other things) after running all shaping, line breaking, and formatting of the text.
-This object **is a snapshot** of metrics given the constraints applied at the time of
-formatting, and is **not updated** as additional changes are made to the data model. In order
-to get new metrics, the `format` function must be run again.
+The `FormattedText` object retains the formatted structure of the input text, and offers APIs to
+query the total bounding box of the text as constrained at any given moment; APIs to adjust the 
+constraints for a given line and its descendents resulting in text that wraps at different 
+specified lengths, and other APIs for serving common use cases for which an understanding of the
+formatted text and its relationship to the input text is important (e.g., editing cases).
+
+This object **is updated** as contraints on the lines are changed. Updates and changes may include
```

Comment:
This paragraph seems contradictory.  The object "is updated" when constraints are changed... but that requires `format` to be called again, which returns a new/different object.

Reply:
This is a holdover from `reflowFrom`, now removed. I suspect something like it will come back (more genericly `reflow` for the whole container, less options, and not available on lines), but I can adjust this text since we don't know if a hypothetical `reflow` would in-place update the object or would instead reaturn a new instance.

Reply:
Mostly I deleted this content and re-wrote some of the prose around it. A more thorough pass will fix other issues when I update the metrics explainer for the glyph-less proposal.